mne/io/tests/test_reference.py

```diff
@@ -308,8 +316,6 @@ def test_add_reference():
     epochs = Epochs(raw, events=events, event_id=1, tmin=-0.2, tmax=0.5,
                     picks=picks_eeg, preload=True, proj='delayed')
     epochs_ref = add_reference_channels(epochs, 'Ref', copy=True)
-    # CAR after custom reference is an Error
-    assert_raises(RuntimeError, epochs_ref.set_eeg_reference)
 
```

Comment:
Can you add a test that after custom ref, you can go to average, and that before/after the info flag changes?